repository-data/local/src/main/resources/hcm-config/configuration/users/unlocker.yaml

```diff
@@ -0,0 +1,8 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:users/unlocker:
+      hipposys:active: true
+      hipposys:password: unlocker
```

Comment:
Does this mean we get a default 'unlocker' user alongside the built-in 'author' and 'editor' in vanilla installations?

What's the impact on already established installations: does such a user get added there on upgrade as well, necessitating a manual intervention to disable it in production environment?

Reply:
not vanilla - localhost only, for testing